apps/src/templates/studioHomepages/Courses.jsx

```diff
@@ -66,11 +67,14 @@ class Courses extends Component {
 
         {/* English, teacher.  (Also can be shown when signed out.) */}
         {(isEnglish && isTeacher) && (
-          <CoursesTeacherEnglish
-            isSignedOut={isSignedOut}
-            showInitialTips={showInitialTips}
-            userId={userId}
-          />
+          <span>
```

Comment:
Why span?  `div` seems more typical.. I think spans are just for inline elements.